src/plugins/colum-chooser/column-chooser.component.ts

```diff
@@ -20,6 +20,12 @@ export class ColumnChooserComponent extends PluginComponent implements OnInit, O
 		super(root);
 
 		this.models = ['columnChooser'];
+
+		this.using(this.model.sceneChanged.on(e => {
```

Comment:
ok, so, can we have a separate FocusService that will do next for now:
1. Injectable to components
2. have a method `focusService.activateAfterRender`